src/pages/home.jsx

```diff
@@ -9,26 +9,45 @@ export default function Home() {
     <h1 className='flexp'>Aktuella filmer på bio</h1>
       <div className="flex-container">
           {movies.map(movie =>
-              <>
-                  <div> <Link to={"/movie/" + movie.title}><img className="topcontainerimages" src={movie.images}
-        width="100px"></img></Link><p>{movie.title}</p></div>      
-              </>
+            <>
+              <div className="topcontainerimagediv">
+                <div className="topcontainerimagedivv">
```

Comment:
där är två divvar som heter nästan likadant ? 
Samt du har tagit bort länken till detalj vyn. 
<Link to={"/movie/" + movie.title}>